site/static/examples/kind-with-registry.sh

```diff
@@ -7,15 +7,15 @@ KIND_CLUSTER_NAME="${KIND_CLUSTER_NAME:-kind}"
 # create registry container unless it already exists
 reg_name='kind-registry'
 reg_port='5000'
-running="$(docker inspect -f '{{.State.Running}}' "${reg_name}")"
+running="$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || :)"
```

Comment:
:+1: thanks
`true` might be a little better known than `:`

Reply:
: changed to true